### PR TITLE
[Android] wallet settings (uplift to 1.33.x)

### PR DESCRIPTION
**Pull request by wchen342**
Manual uplift of https://github.com/brave/brave-core/pull/11034.
Fixes https://github.com/brave/brave-browser/issues/18896.

Pre-approval checklist: 
- [x] You have tested your change on Nightly. 
- [ ] This contains text which needs to be translated. 
    - [ ] There are more than 7 days before the release. 
    - [ ] I've notified folks in #l10n on Slack that translations are needed. 
- [x] The PR milestones match the branch they are landing to. 

Pre-merge checklist: 
- [x] You have checked CI and the builds, lint, and tests all pass or are not related to your PR. 

Post-merge checklist: 
- [x] The associated issue milestone is set to the smallest version that the changes is landed on.

### CODE REVIEW

**Comment by kjozwiak** [comment]
@wchen342 mind taking a look at the above conflict? Looks like we have a conflict with the following file(s):

```
Conflicting files
* browser/ui/android/strings/android_brave_strings.grd
```

**Comment by wchen342** [comment]
@kjozwiak The conflict has been resolved.

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.33.x` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
